src/browser/existing-browser.ts

```diff
@@ -259,13 +259,15 @@ export class ExistingBrowser extends Browser {
         for (const attachToElement of [false, true]) {
             // @ts-expect-error This is a temporary hack to patch wdio's breaking changes.
             session.overwriteCommand("$$", async (origCommand, selector): ChainablePromiseArray<ElementArray> => {
-                    const arr: WebdriverIO.Element[] & { parent?: unknown; foundWith?: unknown; selector?: unknown } =
-                        [];
+                    const arr: WebdriverIO.Element[] & Partial<Pick<ElementArray, "parent" | "foundWith" | "selector" | "props">> = [];
+
                     const res = await origCommand(selector);
                     for await (const el of res) arr.push(el);
+
                     arr.parent = res.parent;
                     arr.foundWith = res.foundWith;
                     arr.selector = res.selector;
+                    arr.props = res.props;
```

Comment:
This field helps to not fail when using `toBeElementsArrayOfSize` command from expect-webdriverio. Fails here - https://github.com/webdriverio/expect-webdriverio/blob/v3.6.0/src/util/formatMessage.ts#L11. So I add `props` to fix it.